pom.xml

```diff
@@ -169,7 +169,8 @@
     <!-- for now, not running scalafmt as part of default verify pipeline -->
     <scalafmt.skip>true</scalafmt.skip>
     <codehaus.jackson.version>1.9.13</codehaus.jackson.version>
-    <fasterxml.jackson.version>2.10.0</fasterxml.jackson.version>
+    <fasterxml.jackson.version>2.10.5</fasterxml.jackson.version>
```

Comment:
We cannot bump up it to 2.12.0?

Reply:
Looking over Jackson's release information both [Jackson Release 2.11](https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.11) and [Jackson Release 2.12](https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.12) showed a large amounts of changes; and although I could attempt a build to make sure compilation fails, I'd personally be uncertain that unit tests would be sufficient enough to ensure compatibility.